javascript/javascript1/week3/main.js

```diff
@@ -0,0 +1,272 @@
+console.log("My username on freeCodeCamp is: Nelia11.")
+
+//Item array removal
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  
+// Write some code here
+for(let i = 0; i < names.length; i++) {
+    if (names[i] === nameToRemove){
+        names.splice(i, 1);
+        i--; // decrement the index variable so it does not skip the next item in the array
```

Comment:
Why do you need to add decrement? Can't you just break out of the loop when you found your match?
Also, can you make it a function?